linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/bash/shared.sh

```diff
@@ -1,3 +1,9 @@
 # platform = multi_platform_all
 
-nmcli radio all off
+if 
+    rpm -qa | grep NetworkManager
```

Comment:
I'm sorry I didn't catch this earlier, could you change this to `rpm -q NetworkManager` ?
It is faster and will match the exact package.